Icon.js

```diff
@@ -23,9 +23,10 @@ class Icon extends Component {
   }
 }
 
+const fontFamily = Platform.OS === 'ios' ? 'FontAwesome' : 'fontawesome-webfont';
```

Comment:
can't we have the same FontAwesome name in both ios and android?

Reply:
That does not seem to work since iOS uses the Font Name and Android uses the filenames. The pull requests amends this so that you can simply copy the files from the FontAwesome4.7.0 Zip. Even renaming does not work for Android as font names should be all lowercase. 

I don't know if that is a recent change, but otherwise it won't work for my react native 0.55.4 projects.